taccsite_cms/static/site_cms/css/src/_imports/elements/html-elements.css

```diff
@@ -63,6 +63,8 @@ main {
 }
 
 h1, h2, h3, h4, h5, h6 {
+  margin-top: revert; /* to overwrite Bootstrap `_reboot.css` */
```

Comment:
**This is they _KEY_ change.** It undoes Bootstrap's `margin-top: 0`, and lets browser defaults apply.